.github/workflows/documentation.yml

```
@@ -27,7 +27,7 @@ jobs:
       - name: Build API documentation
         run: |
           source python-workflow-venv/bin/activate
-          pdoc --output-dir api-documentation src/meshpy/
+          pdoc --math --output-dir api-documentation src/meshpy/
```

While you're at it - I wanted to change it for a long time but I always forgot

```suggestion
          pdoc --math --docformat google --output-dir api-documentation src/meshpy/
```